src/hooks/useLocalStorage/useLocalStorage.ts

```diff
@@ -0,0 +1,50 @@
+import { useCallback, useState } from 'react';
+
+/**
+ * @param key The key of the localStorage.
+ * @param initValue The initial value to set, if value in localStorage is epmpty.
+ * @param {Object} options The options object.
+ * @param {boolean} [options.raw=true] If set true, hook will not attempt to JSON serialize stored values.
+ * @param {Object} [options.serializer=JSON.stringify] Custom serializer.
+ * @param {Object} [options.deserializer=JSON.parse] Custom serializer.
+ * @returns The current value of a localStorage, a callback to set the localStorage and a callback to remove the localStorage.
+ */
+
+export const useLocalStorage = <T>(
```

Comment:
```suggestion
export const useLocalStorage = <T = null>(
```
Тогда будет тип по дефолту если не прокинули `initValue`.